nanoFirmwareFlasher.Tool/Options.cs

```diff
@@ -341,17 +339,51 @@ public class Options
             HelpText = "Reads details from connected device.")]
         public bool DeviceDetails { get; set; }
 
+        [Option(
+            "showfwonly",
+            Required = false,
+            Default = false,
+            HelpText = "Show which firmware to use for a device without deploying anything.")]
+        public bool ShowFirmwareOnly { get; set; }
+
         [Option(
             "filedeployment",
             Required = false,
             Default = null,
             HelpText = "JSON file containing file deployment settings.")]
         public string FileDeployment { get; set; }
 
+        [Option(
```

Comment:
I'm thinking that these options names are too verbose ... can we short them to make it easier to write (and even to read)?
Maybe just drop the fw. Like `archivepath`, `updatearchive` and `fromarchive`. What do you think?

Reply:
I first had planned to use archivepath, but there is also an option backuppath. In my mind archive and backup are quite similar, so I thought it might be confusing. Then I saw that there also is an option --fwversion, so I thought: aha, all firmware-related options have "fw" in the name, let's continue that.

But it is all about explaining what the versions mean. So `archivepath`, `updatearchive` and `fromarchive` are just as good.  I'll change it in the next commit.

Reply:
Good point on the `backuppath`. I don't think this will be confusing.

At the rate all these options are growing (which haven't been foreseeing when it all started) we should do a global revamp on these and start splitting & grouping these into verbs and options. As it is its getting a _bit_ confusing and not very cleverly organized...